src/pytest_regressions/common.py

```diff
@@ -188,3 +189,24 @@ def make_location_message(banner: str, filename: Path, aux_files: List[str]) ->
             else:
                 dump_aux_fn(Path(obtained_filename))
                 raise
+
+
+def round_digits(data: Union[list, dict], prescision: int) -> Union[list, dict]:
```

Comment:
I'm not a native English speaker, but I can't find "prescision" an alternative spelling for "precision". Can you update and use `precision` instead, which seems more common?

Reply:
I agree, but perhaps just `digits` is enough given the function is called `round_digits` already.

Reply:
Ahh, for this function, yes, but for `check` it's better to have the full name :+1:.